tests/test_fields.py

```
@@ -262,16 +262,16 @@ def test_decode_various_logs(self, bitfield, bytes, expected):
     @pytest.mark.parametrize(
         "bytes,expected",
         [
-            (b"\x00\x03", "E_5V_OVER_CURRENT|E_5V_OVER_VOLTAGE"),
-            (b"\x00\x05", "E_5V_OVER_CURRENT|E_5V_UNDER_VOLTAGE"),
+            (b"\x00\x03", "E_5V_OVER_CURR|E_5V_OVER_VOLT"),
+            (b"\x00\x05", "E_5V_OVER_CURR|E_5V_UNDER_VOLT"),
         ],
     )
     def test_decode_from_bytes(self, bitfield, bytes, expected):
```

Ruff (0.15.21)

[error] Function argument `bytes` is shadowing a Python builtin

(A002)